content/remote-development/getting-started.md

```diff
@@ -36,33 +36,40 @@ brew install continuouspipe/tools/cp-remote
 Otherwise you can install it manually via:
 
 ```
-sudo curl https://continuouspipe.github.io/remote-environment-client/0.1.0/darwin-amd64.gz > cp-remote.gz
-gzip -d cp-remote.gz;
+sudo curl https://continuouspipe.github.io/remote-environment-client/latest/darwin-amd64/cp-remote.tar.gz > cp-remote.tar.gz
+tar -xzvf cp-remote.tar.gz;
```

Comment:
Ah that's the "Old Option Style" - which i never heard of until today 😄 
http://www.gnu.org/software/tar/manual/tar.html#subsection

so having the dash is okay
-x extract
-z gzip
-v verbose
-f file target